runtime/etc/opa/opa_conf.yaml

```diff
@@ -4,13 +4,13 @@ services:
     credentials:
       s3_signing:
         metadata_credentials:
-          iam_role: "app-${APPLICATION_ID}"
+          iam_role: "${OPA_IAM_ROLE}"
           aws_region: "${REGION}"
 
 labels:
   cloud_provider: aws
   execution_environment: "${OPA_EXECUTION_ENV}"
-  kio_app_ids: "${APPLICATION_ID}"
+  kio_app_ids: "${KIO_APP_ID}"
```

Comment:
Why does it need to be different from `APPLICATION_ID`?

Reply:
Since there was a concern with the role not always following the pattern app-${APPLICATION_ID}, we thought of being flexible with kio_app_id too. Also in general case nobody has to configure this and it will default to ${APPLICATION_ID}

But if you think, there cannot be a case where an application is using an `APPLICATION_ID` different to their kio app id, I would like to revert this line.

Reply:
I don't see any valid use cases where an instance would have multiple application identities. Let's just use the common configuration item for this and only make it configurable if it's really necessary.